README.md

```diff
@@ -1,2 +1,24 @@
-# akri-wasi
-WebAssembly Systems Interface version for akri
+# Akri and Krustlet integration
+WebAssembly Systems Interface (Wasi) version for akri.
+
+## About
+
+Akri is a Kubernetes component that finds and lets you use IoT devices under the k8s environment. Krustlet is also a Kubernetes component and lets you run WebAssembly application (In addition to docker containers) on your k8s cluster. What we are doing is merging these two ideas together and making Akri compiled as a wasm-wasi application be deployed into a Kubernetes cluster using Krustlet, creating new ways to use these devices in your WASM applications and give Akri users all the advantages of this technology.
+
+As it was designed to be runed on the edge, Akri is not expected to consume lots of memory and could benefit from any improvements in this aspect. By making Akri compatible with Krustlet we expect a significant improvement in startup time, runtime performance and memory consumption, reducing the consumption of the user's resources.
```

Comment:
/sruned/run

Reply:
Would be great to include performance improvements here somewhere (once we know them)